Neos.Fusion/Classes/Core/Cache/ParserCacheFlusher.php

```diff
@@ -50,7 +50,10 @@ public function flushPartialCacheOnFileChanges($fileMonitorIdentifier, array $ch
 
         $identifiersToFlush = [];
         foreach ($changedFiles as $changedFile => $status) {
```

Comment:
What does `$status` tell use? Could this be a way to filter for deleted files?

Reply:
we actually dont want to ignore deleted files as they have to be flushed in order to keep the cache small

Reply:
Not ignore them, but skip the `realpath()` call on them (as that can't be done anyway)

Reply:
This should be fine now with the changed getCacheIdentifierForFile() which no longer does realpath as the filemonitor already returns one anyways.